parts/windows/sendlogs.ps1

```diff
@@ -0,0 +1,38 @@
+<#
```

Comment:
is this used anywhere? maybe break windows to a separate pr?

Reply:
It's not used right now, but including it in the build would mean that the windows team could use it manually. Instead of having to have support run collect-aks-logs.ps1 and then pull it from the host to send, they could stop the waagent and run send-logs.py to force the upload to the host where support can get it. If you want it in a separate PR we can pull it out, but I think it's useful to include even before they integrate it in their provisioning script.